src/components/DocumentSelector/index.js

```diff
@@ -1,15 +1,11 @@
-import { h, Component } from 'preact'
-import { kebabCase } from '~utils/string'
-import { isEmpty } from '~utils/object'
+import {Component} from 'preact'
```

Comment:
Have we change the spacing eslint rule here?

Reply:
IDE was not configured to expected style settings. I run `prettier` and now it should be okay. Please focus on the rest of the PR.